docs/python/tutorials/step-tutorial.md

```diff
@@ -45,4 +45,21 @@ For example, we configure this step to retry exceptions (such as if `example.com
 @DBOS.step(retries_allowed=True, max_attempts=10)
 def example_step():
     return requests.get("https://example.com").text
+```
+
+
+### Coroutine Steps
+
+You can optionally define DBOS steps as [coroutine funcitons](https://docs.python.org/3/glossary.html#term-coroutine-function).
```

Comment:
```suggestion
You can optionally define DBOS steps as [coroutine funcitons](https://docs.python.org/3/glossary.html#term-coroutine-function).
```
You may also decorate async Python functions (functions defined with `async def`, also called coroutines) with `@DBOS.step` to make them DBOS steps.